examples/angular/src/BUILD.bazel

```diff
@@ -237,21 +243,20 @@ container_image(
 
 ts_library(
     name = "universal_server_lib",
-    srcs = [
-        "server.ts",
-    ],
+    srcs = ["server.ts"],
     deps = [
         "//src/app:app_server",
         "@npm//@nguniversal/express-engine",
         "@npm//@types/node",
+        "@npm//compression",
```

Comment:
also this

Reply:
I think my brain was half-asleep this morning :man_facepalming: